configuration/wireless/wireless-cli.md

```diff
@@ -34,6 +34,16 @@ network={
     psk="testingPassword"
 }
 ```
+**NOTE: Some cards requires your ISO 3166 country code.**  
```

Comment:
Not sure I understand what "some cards requires.." means here.

Reply:
Me neither tbh, i just recognize that some of mine wont work without the country parameter set.

Reply:
Does it need rephrasing? "Some USB wireless adapters"? Or something else?

Reply:
Looking furthur down the page, there is a section on 3B+ and the requriements there for the country code to be set. I wonder if we simply add "for 3B+ and any USB wireless device using 5gig" or similar to that section rather than have this extra section? Or have people found its is necessary for other protocols as well?

Reply:
I don't think it affects USB devices.

Reply:
In any case, "Some cards may require" is probably better than "Some cards requires" ? :wink: